tests/test_fields.yml

```diff
@@ -161,8 +161,10 @@ components:
           minimum: -1000
         field_file:
           type: string
+          format: binary
```

Comment:
This is a byproduct of using DRF's `map_field`.  I can avoid it if it is problematic.

Reply:
Diagnosed and raised as https://github.com/tfranzel/drf-spectacular/pull/69